### PR TITLE
Merge v0.13 updates to v0.10

### PR DESCRIPTION
**Pull request by MacSplody**

### CODE REVIEW

**Comment by MacSplody** [comment]
I'd prefer someone give it a review since its a large commit. If your happy enough with that it can be merged.

File: masterlist.yaml [hunk 1]
```diff
@@ -198,7 +203,7 @@ globals:
 # SKSE64 plugins requirement
   - <<: *requireSKSE64
     subs: [ 'a SKSE Plugin' ]
-    condition: 'file("SKSE/Plugins/([^\.]+\.dll)") and not file("../skse64_loader.exe")'
+    condition: 'file("SKSE/Plugins/([^\.]+\.dll)") and not file("../skse64_loader.exe") and not file("../sksevr_loader.exe")'
```
Comment (MacSplody) [inline masterlist.yaml:206]: Are the VR checks even required?

File: masterlist.yaml [hunk 2]
```diff
@@ -324,6 +356,14 @@ plugins:
         itm: 69
         udr: 8
         nav: 1
+    clean:
+      # Version 1.59.39.0 - Auto Clean
+      - crc: 0x5eb9eade
+        util: SSEEdit v3.2.1
+  - name: 'SkyrimVR.esm'
+    global_priority: -127
```
Comment (MacSplody) [inline masterlist.yaml:364]: VR again, remove?

File: masterlist.yaml (hunk 1 above)
Comment (Ortham) [reply masterlist.yaml:206]: Yes, if it's not there then SKSEVR users will see messages about not having SKSE installed.

File: masterlist.yaml (hunk 2 above)
Comment (Ortham) [reply masterlist.yaml:364]: Nope, Skyrim VR uses the same masterlist because there's so little difference, and `SkyrimVR.esm` needs to load early.

**Review by Ortham — APPROVED** [review]
Looks good to me.

**Comment by MacSplody** [comment]
Okay, thanks.

**Pull request merged**